### PR TITLE
Create gravitationalForceEquation.java

### PR DESCRIPTION
**Pull request by zazulam**
This method is meant to calculate the gravitational force, F = (G*Mm)/(R^2), between two objects rather than the applied force of an object i.e. F=ma.

### CODE REVIEW

**Comment by BaReinhard** [comment]
If you can change the path name to match the new file structure in the contributions.md I'll merge this.

File: java/physics/force/gravitational_force/gravitational_force_equation.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+Public class gravitational_force_equation{
+	//Meant to calculate teh gravitational force between two objects
```
Comment (mdew192837) [inline java/physics/force/gravitational_force/gravitational_force_equation.java]: Typo

Comment (mdew192837) [inline java/physics/force/gravitational_force/gravitational_force_equation.java]: Space this word

File: java/physics/force/gravitational_force/gravitational_force_equation.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+	//There exists the gravitational constant of the universe G, 6.67 * 10^-11
```
Comment (mdew192837) [inline java/physics/force/gravitational_force/gravitational_force_equation.java]: Ditto from above

File: java/physics/force/gravitational_force/gravitational_force_equation.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+        
+	public static double final g = .0000000000667;
+        
+        /*@param m1 corresponds to the mass of the first object in kgs
+         *@param m2 corresponds to the mass of the second object in kgs
+         *@param r corresponds to the distance between both masses in meters
+         */
+
```
Comment (mdew192837) [inline java/physics/force/gravitational_force/gravitational_force_equation.java]: No lines here

**Pull request merged**